src/base/ScillaParser.mly

```diff
@@ -233,6 +233,7 @@ targ:
 | LPAREN; t = typ; RPAREN; { t }
 | d = scid; { to_type d (toLoc $startpos(d))}
 | t = TID; { TypeVar t }
+| t = address_typ; { t }
```

Comment:
I guess I missed where the `address_typ` production was introduced.

Reply:
It's in the remote_state_reads branch, which this PR is to be merge into.